pages/Footer.js

```diff
@@ -0,0 +1,58 @@
+import 'react-native-gesture-handler';
+import React from 'react';
+import { View, Linking, StyleSheet } from 'react-native';
+import FooterButton from '../components/FooterButton.js';
+
+
+const styles = StyleSheet.create({
+    styledFooter: {
+        height: '100%',
+        width: '100%',
+        display: 'flex',
+        flexDirection: 'column',
+        justifyContent: 'space-between',
+    },
+
+});
+
+export const Footer = ({ navigation }) => {
+    return (
+
+        <View style={styles.styledFooter}>
+            <FooterButton
+                onPress={() => navigation.navigate('About')}
+                source={require("../assets/genconnect_footerbuttons_about.png")}
+            />
+            <FooterButton
+                source={require("../assets/genconnect_footerbuttons_share.png")}
+                onPress={() =>
+                    Linking.openURL(
+                        'https://apps.apple.com/us/app/gen-connect/id1588977864'
+                    )
+                }
+            />
+            <FooterButton
+                source={require("../assets/genconnect_footerbuttons_review.png")}
+                onPress={() =>
+                    Linking.openURL(
+                        'https://apps.apple.com/us/app/gen-connect/id1588977864'
+                    )
+                }
+            />
+            <FooterButton
+                onPress={() => Linking.openURL('mailto:quyionah@coolmomsdancetoo.com')}
+                source={require("../assets/genconnect_footerbuttons_contact.png")}
+            />
+            <FooterButton
+                onPress={() =>
+                    Linking.openURL(
+                        'https://www.coolmomsdancetoo.com/'
+                    )
+                }
+                source={require("../assets/genconnect_footerbuttons_feedback.png")}
+            />
```

Comment:
why is the feedback linked to the cmdt website? what's the diff between feedback and review?

Reply:
Will be changing where the link goes. Review goes to the app store and feedback is a google survey created by Quyionah to get more app-specific info.